tests/python/mkl/test_mkldnn.py

```diff
@@ -240,5 +240,135 @@ def check_batchnorm_training(stype):
     for stype in stypes:
         check_batchnorm_training(stype)
 
+@with_seed()
```

Comment:
I don't think that these tests are mkldnn specific and should rather be but in test_operators or test_operators_sparse
@eric-haibin-lin @haojin2

Reply:
Right, but these tests can then be placed in the regular operator tests. The fallback will be implicitly tested. I don't see any mkldnn specific code that would justify it from not being run with MXNet or cuda as backend.

I had the same discussion in another PR from shufan (on mobile ATM). Would you mind joining in over there?

Reply:
I definitely would like to, would you mind providing a pointer to that PR?

Reply:
https://github.com/apache/incubator-mxnet/pull/10921

Reply:
Thanks